src/qibolab/platforms/platform.py

```diff
@@ -30,14 +30,37 @@ def disconnect(self):
         self.design.disconnect()
         self.is_connected = False
 
-    def execute_pulse_sequence(self, sequence, nshots=1024, relaxation_time=None):
+    def execute_pulse_sequence(
+        self, sequence, nshots=1024, relaxation_time=None, fast_reset=False, sim_time=10e-6, acquisition_type=None
+    ):
         if relaxation_time is None:
             relaxation_time = self.relaxation_time
-        return self.design.play(self.qubits, sequence, nshots=nshots, relaxation_time=relaxation_time)
+        if fast_reset is True:
+            fast_reset = {}
+            for qubit in self.qubits.values():
+                if "c" in str(qubit.name):
+                    pass
+                else:
+                    fast_reset[qubit.name] = self.create_RX_pulse(qubit=qubit.name, start=0)
```

Comment:
Since you're just creating a dictionary, consider using dictionary comprehension:
```suggestion
            names = (qubit.name for qubit in self.qubits.values())
            fast_reset = {
                name: self.create_RX_pulse(qubit=name, start=0)
                for name in names if "c" not in str(name)
            }
```
(formatting is bad, I'm writing in GitHub, Black knows how to do it properly)

Reply:
Just changed it a bit following the changes from the last comment.